src/app/models/Log.tsx

```diff
@@ -21,98 +21,53 @@ export interface Log {
 }
 
 const LogModel = {
-    list: [] as any[],
-    current: {} as Log,
-    createLog: {} as Log, // log being created
-    async fetch(query?: string) {
-        return m.request({
-            method: 'GET',
-            url: `${API_URL}logs${query ? `?${query}` : ''}`,
-            withCredentials: false
-        }).then((result: any) => {
-            this.list = result;
-        }).catch(() => {
-            const result = dummyLogList;
-            this.list = result;
-        });
+    list: {
+        error: '',
+        logs: [] as any[],
+        async fetch(query?: string) {
+            return m.request({
+                method: 'GET',
+                url: `${API_URL}logs${query ? `?${query}` : ''}`,
+                withCredentials: false
+            }).then((result: any) => {
+                LogModel.list.logs = result;
+            }).catch((e: any) => {
+                LogModel.list.error = e.message;
+            });
+        }
     },
-    async fetchOne(id: number) {
-        return m.request({
-            method: 'GET',
-            url: `${API_URL}logs/${id}`,
-            withCredentials: false
-        }).then((result: any) => {
-            this.current = result;
-        }).catch(() => {
-            const result = dummyLogList[0];
-            this.current = result;
-        });
+    current: {
+        error: '',
+        log: {} as Log,
+        async fetchOne(id: number) {
+            return m.request({
+                method: 'GET',
+                url: `${API_URL}logs/${id}`,
+                withCredentials: false
+            }).then((result: any) => {
+                LogModel.current.log = result;
+            }).catch((e: any) => {
+                LogModel.current.error = e.message;
+            });
+        }
     },
-    save() {
-        LogModel.createLog.creationTime = new Date().toString();
-        LogModel.createLog.origin = 'human';
-        return m.request<Log>({
-            method: 'POST',
-            url: `${API_URL}logs`,
-            data: LogModel.createLog,
-            withCredentials: false
-        });
+    createLog: {
+        error: '',
+        log: {} as Log,
+        async save() {
+            LogModel.createLog.log.creationTime = new Date().toString();
+            LogModel.createLog.log.origin = 'human';
```

Comment:
Should it be the same this?

Reply:
I am not sure what you mean here?

Reply:
The same as with creationTime, that it should be in the api instead of fe?